### PR TITLE
chore(commands): Update documentation about how to develop che-dashboard

### PR DESCRIPTION
**Pull request by benoitf**
### What does this PR do?
Update documentation on how to develop against a remote server
Also remove che.openshift.io reference as it has been deleted

### What issues does this PR fix or reference?
N/A

#### Release Notes
N/A

#### Docs PR
N/A

### CODE REVIEW

**Review by akurinnoy — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -32,19 +32,45 @@ yarn
...
+The start command requires to specify a remote Eclipse Che server like:
 ```sh
-yarn start
+yarn start --env.server=https://che-che.192.168.99.100.nip.io
 ```
 
 The development server serves the project on [http://localhost:3000](http://localhost:3000).
-By default it proxies all API requests to [che.openshift.io](https://che.openshift.io). You can change this behavior providing your own proxy target url and port using the following command as an example:
+
+note: To use Eclise Che Hosted by Red Hat instance at https://workspaces.openshift.com, use the fully qualified host name of the cluster.
```
Comment (sleshchenko) [inline README.md:42]: technically it's not Eclipse Che
```suggestion
note: To use CodeReady Workspaces(based on Che) Hosted by Red Hat instance at https://workspaces.openshift.com, use the fully qualified host name of the cluster.
```

**Review by sleshchenko — APPROVED** [review]

**Comment by benoitf** [comment]
is it expected to have all PR/docker build failing ?

File: README.md (hunk 1 above)
Comment (benoitf) [reply README.md]: yep thanks

**Comment by sleshchenko** [comment]
> is it expected to have all PR/docker build failing ?

yes. it's created from fork, while job is configured to login to use cache images.
As quick fix we may just remove caching, since I believe it does not work properly at this point.

**Pull request merged**